core/src/main/res/values/styles.xml

```diff
@@ -62,7 +62,7 @@
     <item name="android:gravity">end</item>
   </style>
 
-  <style name="no_content_text" parent="TextAppearance.KiwixTheme.Headline5" />
+  <style name="no_content_text" parent="TextAppearance.KiwixTheme.Headline6" />
```

Comment:
this makes me suspicious that our styles are not at the right hierarchy. 
We have 
```
  <style name="no_list_content_text" parent="TextAppearance.KiwixTheme.Headline6">
    <item name="android:layout_width">wrap_content</item>
    <item name="android:layout_height">wrap_content</item>
    <item name="android:visibility">gone</item>
    <item name="layout_constraintLeft_toLeftOf">parent</item>
    <item name="layout_constraintRight_toRightOf">parent</item>
    <item name="layout_constraintVertical_bias">0.45</item>
    <item name="layout_constraintTop_toTopOf">parent</item>
    <item name="layout_constraintBottom_toBottomOf">parent</item>
  </style>
```
which is used in download/library

and
```
  <style name="no_content_text" parent="TextAppearance.KiwixTheme.Headline6" />
```
which is used in reader/search

it seems to me that we should either have 1 style that suits all of these cases or a chain of inheritance as all this `no_content` stuff is clearly related

Reply:
Yes... I will do it